### DISCUSSION TITLE
Fieldset

### DISCUSSION THREAD

**Discussion by bameyrick** [ONSdigital/design-system#1380] [Feedback]
Use this place to discuss the [fieldset component](https://ons-design-system.netlify.com/components/fieldset) in the [ONS Design System](https://ons-design-system.netlify.com)

**Comment by jrbarnes9** [comment]
Fieldset config table refers to 'field' rather than 'fieldset' in places

<img width="934" alt="Screenshot 2020-09-03 at 11 18 47" src="https://user-images.githubusercontent.com/43346934/92103122-69fa8a00-edd7-11ea-9789-bd326e35a173.png">

**Reply by jrbarnes9** [reply]
#955 resolved

**Comment by jrbarnes9** [comment]
Remove bottom margin on last field in a fieldset to allow control over fieldset vertical spacing

<img width="663" alt="Screenshot 2020-09-03 at 11 23 01" src="https://user-images.githubusercontent.com/43346934/92103536-002eb000-edd8-11ea-94ee-8389bf60dc01.png">

**Reply by jrbarnes9** [reply]
#956 resolved

**Comment by jrbarnes9** [comment]
Change name of `legendIsPageTitle` to avoid confusion with page `<title>` which can differ to H1 heading

Change param to perhaps `legendIsQuestionTitle` or `legendIsPageHeading`

**Reply by jrbarnes9** [reply]
Resolved #1575

**Comment by jrbarnes9** [comment]
The `<div>` for `ons-fieldset__description` is invalid HTML when nested within the `<legend>` when using `params.legendIsQuestionTitle`
